lib/kbsecret/session.rb

```diff
@@ -80,6 +80,17 @@ def add_record(type, label, *args)
       record.sync!
     end
 
+    # Load an existing record from another session
+    # @param record [Record] an instance of a record
+    # @return [void]
+    def import_record(record)
+      raise Exceptions::RecordDuplicationError.new(self, record.session) if self == record.session
```

Comment:
Thanks for adding this. Make sure to actually define this class in `lib/kbsecret/exceptions.rb`, too.

Reply:
Done